src/demo/errors.py

```diff
@@ -0,0 +1,34 @@
+from judgeval.tracer import Tracer
```

Comment:
Couple notes:
1. How are you storing the error results in the database? It'd be great to have a dedicated field for this so we can easily aggregate stats for dashboard purposes. Talk to @alanzhang25 about this!
2. More of a frontend design question - I want to brainstorm the best way to indicate an error occurred on the span/Trace itself.